src/abi/OlympusGiving.json

```diff
@@ -1,321 +1,3 @@
-{

```

Comment:
@0xLienid deliberate removal?

Reply:
yeah, I replaced this abi but just didn't format it yet